packages/ui/src/components/accordion/__tests__/accordion.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react'
+
+import '@testing-library/jest-dom'
+
+import { Accordion } from '../accordion'
+
+describe('Accordion Component', () => {
+  test('renders the accordion with trigger and content', () => {
+    render(
```

Comment:
better way to do it:

```suggestion
    const { getByText } = render(
```

Reply:
Could be better to use screen. https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen